templates/docker-rotate

```diff
@@ -4,4 +4,4 @@
 docker-rotate images --keep 3
 # Remove docker containers that have exited over an hour ago
 # and were created over a day ago
-docker-rotate containers --exited 1h --created 1d
+docker-rotate containers --exited 1h --created 1d --images {{ docker_rotate_containers_images_filter }}
```

Comment:
docker_rotate_container_images_regex maybe?  The "exclude" part is misleading because the only reason the busybox images are excluded is because of the "~" prefix.  Originally I used an inclusive filter of just "locationlabs" ...